certification/internal/engine/openshift.go

```diff
@@ -15,13 +16,26 @@ import (
 	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/cli"
 	client "github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/client"
+	"k8s.io/client-go/tools/clientcmd"
 )
 
-type OpenshiftEngine struct {
+type openshiftEngine struct {
 	KubeConfig *rest.Config
 }
 
-func (pe OpenshiftEngine) CreateNamespace(name string, opts cli.OpenshiftOptions) (*cli.OpenshiftReport, error) {
+func NewOpenshiftEngine() *cli.OpenshiftEngine {
+	k8sconfig, err := clientcmd.BuildConfigFromFlags("", os.Getenv("KUBECONFIG"))
```

Comment:
It appears that this is causing an issue in the unit tests. This will have to be done elsewhere, or don't use this in the unit test and just instantiate the openshiftEngine directly.